week-01-controlling-the-dom/day-03-js/dusk-arrays/README.md

```diff
@@ -256,102 +256,5 @@ fruits.forEach(function(element, index) {
 */
 ```
 
-###array.map()###
```

Comment:
I removed these from week01 so that future cohorts don't expect that they can cover all of this material in one day.  We're using the exact same stuff in week 2)